stackset.tf

```diff
@@ -11,8 +11,8 @@ resource "aws_cloudformation_stack_set" "permeate_account_policy" {
   }
 
   parameters = {
-    ExpelCustomerOrganizationGUID = var.expel_customer_organization_guid,
-    ExpelAssumeRoleARN            = aws_iam_role.expel_assume_role.arn
+    ExpelCustomerOrganizationGUID = var.expel_customer_organization_guid
+    ExpelAccountARN               = var.expel_aws_account_arn
     ExpelRoleName                 = aws_iam_role.expel_assume_role.name
```

Comment:
nit: should this be set from variables as well?

Reply:
I would say no there since it's dependent on the role being created anyway. Better to just pull it from the created resource.